exonum/src/api/node/public/system.rs

```diff
@@ -188,7 +216,22 @@ impl SystemApi {
             .handle_healthcheck_info("v1/healthcheck", api_scope)
             .handle_user_agent_info("v1/user_agent", api_scope)
             .handle_list_services_info("v1/services", api_scope)
-            .handle_proto_source("v1/proto-sources", api_scope);
+            .handle_proto_source_core("v1/proto-sources/core", api_scope)
+            .handle_proto_source_service("v1/proto-sources/service", api_scope);
         api_scope
     }
 }
+
+fn filter_exonum_sources(files: Vec<ProtoSourceFile>) -> Vec<ProtoSourceFile> {
+    let sources = EXONUM_PROTO_SOURCES
```

Comment:
Do I understand it correctly that `EXONUM_INCLUDES` will not be filtered?

Reply:
They filtered during the build phase.

Reply:
Well... That's kinda not obvious. Includes for exonum are filtered from service includes during build phase, but exonum files itself are not filtered. Do I understand that correctly?